perfact/zodbsync/zodbsync.py

```diff
@@ -278,7 +278,7 @@ def create_manager_user(self):
         Make sure the manager user exists.
         """
         userfolder = self.app.acl_users
-        user = userfolder.getUserById(self.manager_user)
+        user = userfolder.getUser(self.manager_user)
```

Comment:
Why is it that there's no wrapping happening here? All other `.getUser()` calls have the wrapper.

Reply:
Ah, maybe because `user` is only used for a test.

Reply:
Exactly. `None` has no attribute `__of__`, but we only want to check for `None` here.
BTW, `getUser` and `getUserById` seem to do the same.

Reply:
Yes, it's quite clear. I should have read the following line a bit more attentively.